index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Sergio Usma's portfolio</title>
+  </head>
+  <body>
+    <div class="header">
+      <header>
+        <nav>
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+
+          <div class="burger">
+            <div class="burger-line top"></div>
+            <div class="burger-line mid"></div>
+            <div class="burger-line bot"></div>
+          </div>
+        </nav>
+      </header>
+    </div>
+
+    <section id="main">
+      <div class="main">
+        <h2>
+          Hey there. <br> I'm Sergio. <br />
+          <span>I'm a software <br>developer</span>
+        </h2>
```

Comment:
- [ ] Please consider using `h1` tag for the main heading ⚒ The `h1` tag is usually used for the title of a page. It is an important signal that lets search engines know what your page is about. Check out [this link](https://www.reliablesoft.net/h1-tag/) for more information. 🔗